src/ontology/mondo-edit.obo

```diff
@@ -211929,11 +211929,11 @@ xref: ICD9:752.49 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: OMIM:277000 {source="Orphanet:247775", source="MONDO:equivalentTo", source="Orphanet:247775/e", source="GARD:0007100"}
 xref: Orphanet:247775 {source="OMIM:277000", source="MONDO:equivalentTo"}
 xref: Orphanet:3109 {source="OMIM:277000"}
-xref: SCTID:8793008 {source="MONDO:equivalentTo"}
 xref: UMLS:C0431637 {source="MONDO:notFoundInDiseaseSubset", source="GARD:0007100"}
 xref: UMLS:C0431648 {source="Orphanet:247775", source="MONDO:notFoundInDiseaseSubset"}
```

Comment:
Since `UMLS:C0431648` was added to the parent `MONDO:0017771` in this PR, should this `xref` be removed _or_ wait for the other sources and our pipelines to make this change? 

A few lines up, `xref: SCTID:8793008 {source="MONDO:equivalentTo"}` is removed as it's also now on the parent class, but it also does not have an external source reference?

Reply:
`UMLS:C0431648` : We keep these as a record. What the record says is that (at one point) Orphanet:247775 said that this Mondo term mapped to this UMLS (this information come from the beginning of Mondo creation). But when we reviewed it, we decided that it was not a `Mondo:equivalentTo`. So we keep the "old" or "incorrect" x-ref that came from external sources to keep as a record. We have had a lot of back and forth discussions about whether or not keeping these is useful (and we haven't made a decision yet).

`xref: SCTID:8793008 {source="MONDO:equivalentTo"}`
If i understand, you are asking why this xref only has a MONDO:equivalentTo, and not a source that is an orphanet or omim or gard or... ID). 
Having the external source indicates that the x-ref (here, SCTID) mapping to the Mondo term came from the external source. Since no external sources said that SCTID:8793008 was equivalent to the Mondo term, there are no sources for SCTID:8793008.

Reply:
Thanks! It is confusing to see `UMLS:C0431648` still on this more specific term in this PR when this xref is being added to the parent term.